test/splitgraph/commandline/http_fixtures.py

```diff
@@ -1,6 +1,7 @@
 import json
 
-from splitgraph.cloud import _PROFILE_UPSERT_QUERY
+from splitgraph.cloud import _PROFILE_UPSERT_QUERY, _BULK_UPSERT_REPO_PROFILES_QUERY, _BULK_UPDATE_REPO_SOURCES_QUERY, \
+    _BULK_UPSERT_REPO_TOPICS_QUERY
```

Comment:
Can you make sure you've run the formatter for this file (there's pre-commit hooks for that, you can also just run `black  test/splitgraph/commandline/http_fixtures.py`

Reply:
(i misclicked in GH's review GUI -- this is the second actionable here apart from some user-friendly output)

Reply:
Certainly, will check it.